### ISSUE TITLE
PopupProps.modalBottomSheet hides when keyboard opens up

### ISSUE THREAD

**Issue by akabisdev** [salim-lachdhaf/searchable_dropdown#650]
**Describe the bug**
Using dropdown_search version 5.0.6, with flutter sdk version 3.22.2, using PopupProps.modalBottomSheet, when bottomsheet is visible to the user, and user tries to enter query in search box, keyboard opens up and hides the bottom sheet if constraints are set.

**To Reproduce**
This is happening with version 5.0.6, earlier it was working fine, but version 5.0.5 is not compatible with flutter sdk version 3.22.2

**Expected behavior**
When keyboard opens up, it should not hide bottom sheet

**Screenshots**
<img width="309" alt="Screenshot 2024-06-24 at 11 14 06 AM" src="https://github.com/salim-lachdhaf/searchable_dropdown/assets/84234310/03dd40ee-fe66-41ac-89fc-24c43265cbe6">
<img width="352" alt="Screenshot 2024-06-24 at 11 14 37 AM" src="https://github.com/salim-lachdhaf/searchable_dropdown/assets/84234310/e78719c1-7778-4532-9ca5-fa031cb0f375">

**Smartphone (please complete the following information):**
 - Device: Android Emulator, API 34
 - OS: Android 14
 - Version : Plugin version 5.0.6, Flutter version 3.22.2

**Comment by KD-Dhiren** [comment]
any fixes on this ?

**Comment by glennangelo** [comment]
Adding this property to the parent scaffold of the widget solves this problem for me on 5.0.6

```
Scaffold(
    resizeToAvoidBottomInset: false,  
```
          
and my modalBottomSheetProps

```
isScrollControlled: true,
padding: EdgeInsets.only(bottom: MediaQuery.of(context).viewInsets.bottom),
useSafeArea: false,
```

**Comment by dhiren-basra** [comment]
Hi @glennangelo
I tried as per your suggestion : which not works, bottomSheet keep sticky at bottom, and not moving up.
```
Scaffold(
    resizeToAvoidBottomInset: false,  
```

also where to add this properties, 
```
isScrollControlled: true,
padding: EdgeInsets.only(bottom: MediaQuery.of(context).viewInsets.bottom),
useSafeArea: false,
```
above properties not available in below code, also not in scrollbarProps ..
`popupProps: PopupProps.modalBottomSheet(
 scrollbarProps: ScrollbarProps(),

)`


using version 5.0.6

**Comment by salim-lachdhaf** [comment]
fixed in the next version v6.0.0-pre that will be released in next days
